documentation/modules/exploit/multi/http/wp_responsive_thumbnail_slider_upload.md

```diff
@@ -0,0 +1,51 @@
+
+## Vulnerable Application
+
+  This module exploits an arbitrary file upload vulnerability in Responsive Thumbnail Slider Plugin v1.0 for WordPress post authentication. 
```

Comment:
Add a link on the plug-in to where to download it ( exploit-db has it. Format is ```[Responsive Thumbnail Slider Plugin v1.0](https://www.exploit-db.com/apps/f5d34e16d07e61ad6826d2c1f3d16089-wp-responsive-thumbnail-slider.zip)```